### DISCUSSION TITLE
Git Storage problem

### DISCUSSION THREAD

**Discussion by Techeuroadmin** [requarks/wiki#7249] [Error / Bug Report]
Please, could someone help me fix a problem with storing using Git? I'm trying to use SSH keys; the key is already working, but when I push in WikiJS, I encounter a strange error. My issue is:

Error: spawn ./data/repo EACCES at ChildProcess._handle.onexit (node:internal/child_process:284:19) at onErrorNT (node:internal/child_process:477:16) at process.processTicksAndRejections (node:internal/process/task_queues:82:21)

**Comment by BartWojda** [comment]
On the storage page under Git>SSH Private Key Mode do you have contents selected. I've had a similar issue some time ago and changing this dropdown to contents fix it.
